numpy/random/mtrand/mtrand.pyx

```diff
@@ -713,10 +735,11 @@ cdef class RandomState:
 
         Parameters
         ----------
-        state : tuple(str, ndarray of 624 uints, int, int, float)
+        state : tuple(tuple(int, str), ndarray of 624 uints, int, int, float)
             The `state` tuple has the following items:
 
-            1. the string 'MT19937', specifying the Mersenne Twister algorithm.
+            1. the version of the RNG methods, and the string 'MT19937',
+               specifying the Mersenne Twister algorithm.
```

Comment:
The other options (that I can see) are:
- put the version last (ugh, that doesn't even make sense, the last two entries (`has_gauss` and `cached_gaussian`, i.e. Box-Muller info I believe) should relate to the methods version, not to the underlying RNG, so I'd rather have the version info come before.
- add an extra entry either before or after `"MT19937"` (this doesn't seem very nice either).
- move to a completely new scheme, e.g. `(RNG, version, RNGstate, versionstate)`.